applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/child_image/mcuboot/prj.conf

```diff
@@ -25,6 +25,7 @@ CONFIG_BOOT_SIGNATURE_KEY_FILE="mcuboot_private.pem"
 
 # Flash
 CONFIG_FLASH=y
+CONFIG_SPI_NOR=n
```

Comment:
this commit (57805dbf95b3c66bcbd764f6bf32b055879ff989) can be dropped after we picked the right commit from upstream:
https://github.com/zephyrproject-rtos/zephyr/pull/71425/commits/0713655f830d04f5a647e72815b564a3135335ce
which has the node disabled.

Reply:
dropped